examples/simulator-own-addr/index.rsh

```diff
@@ -0,0 +1,33 @@
+'reach 0.1';
+'use strict';
+
+export const main =
+  Reach.App(
```

Comment:
You should not commit new code with the ternary syntax

Reply:
Or is this just a copy of something else? If so, you could do the thing where you copy and example into this directory.